modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,193 @@
+It provides a foundation based on new technologies and officially supersedes the former `gallery` and `files_videoplayer` apps.
+ownCloud Server 10.3 does not bundle `gallery` and `files_videoplayer` anymore. 
+Instead, it bundles `files_mediaviewer`. 
+With this change, support and maintenance for `gallery` and `files_videoplayer` are discontinued. 
+More details on the Media Viewer can be found in the https://owncloud.org/news/good-bye-gallery-say-hi-to-the-new-media-viewer/[release blog post].
+
+* For a clean transition to Media Viewer, it is necessary to **disable both deprecated apps before the upgrade** using either the admin "Apps" panel in the web interface or via `occ` (e.g., `occ app:disable gallery files_videoplayer`). 
+* After the upgrade, enable the Media Viewer app via the admin panel or `occ app:enable files_mediaviewer`.
+* It is not recommended to continue with the deprecated apps. 
+  However, if you want to do so, you can copy over the `files_videoplayer` directory from the `apps/` folder of the previous ownCloud Server directory and obtain `gallery` from the https://marketplace.owncloud.com/apps/gallery[ownCloud Marketplace].
+
+TIP: Please do not enable `gallery`/`files_videoplayer` and `files_mediaviewer` simultaneously, as these apps are mutually exclusive.
+
+For more information on the Media Viewer app, visit the https://doc.owncloud.com/server/admin_manual/installation/apps/mediaviewer/[ownCloud Documentation].
+
+=== OAuth2 and session handling improvements
+
+Server 10.3 comes with improvements for session handling with Redis.
+These are designed to cope with issues encountered around duplicate session tokens, which make the ownCloud Clients lose their OAuth2 authorization from time to time, and force users to re-authenticate.
+
+The session handling in ownCloud 10.3.0 has been generally improved, making user and client sessions more stable. 
+If Redis is used for session handling, it is necessary to enable Session Locking to ensure that the mentioned issues no longer occur. 
+
+TIP: You can find out if Redis session handling is configured in your web server if you generate an ownCloud Configreport in the web interface. You will find the value `session.save_handler` set to `redis`. 
+
+* It is recommended to use Redis Session Locking if Redis is used for session handling (minimum required version for `php-redis` is `4.1.0`)
+* Enable Redis Session Locking by setting `redis.session.locking_enabled = 1` in `php.ini`
+
+TIP: If Redis is just used as a memory cache or not in use at all, you do not have to apply changes.
+
+TIP: Please note that Redis Session Locking is not supported in clustered Redis environments.
+
+TIP: If your ownCloud deployment is based on the official Docker images or the Univention appliance, you do not have to apply changes as Redis is not used for session handling (unless you configured it differently using ENV variables).
+
+// TODO
+**Docs?**
```

Comment:
There are still these `TODO` lines here - is there still something to do? Or delete these lines?

Reply:
There should only be one left. They're Asciidoc comments, so they won't appear in the published documentation.

Reply:
Just removed that one, as it doesn't appear to be necessary.